src/binder/statement/select_statement.cpp

```diff
@@ -204,6 +204,7 @@ auto SelectStatement::ResolveColumn(const std::string &col_name) -> std::unique_
   switch (table_->type_) {
     case TableReferenceType::BASE_TABLE: {
       auto base_table_ref = dynamic_cast<BoundBaseTableRef *>(table_.get());
+      // TODO(chi): handle case-insensitive table / column names
```

Comment:
The reason here is Postgres parser will automatically convert all identifiers to lower-case. But our tables are generated with programs instead of SQLs, so their table names are upper-lower-mixed. We'll need to convert all table names stored internally to lowercase when binding the parse tree.

Reply:
ah, sorry, I was going to mark this as resolved / it was a cheap jab at mysql